### PR TITLE
feat: typographic reduction and spacing updates

### PR DESCRIPTION
**Pull request by asangma**
**Related Issue:** #886 

## Summary
Broad updates to the typography and spacing in all components.

### CODE REVIEW

File: src/components/calcite-shell-panel/calcite-shell-panel.scss [hunk 1]
```diff
@@ -26,8 +26,8 @@
   min-width: var(--calcite-app-shell-panel-min-width);
   max-width: var(--calcite-app-shell-panel-max-width);
   min-height: var(--calcite-app-shell-panel-min-height);
-  border-left: 1px solid var(--calcite-app-border);
-  border-right: 1px solid var(--calcite-app-border);
+  // border-left: 1px solid var(--calcite-app-border);
```
Comment (driskull) [inline src/components/calcite-shell-panel/calcite-shell-panel.scss:29]: remove comments

File: src/demos/shell/demo-app-temp.css [hunk 2]
```diff
@@ -0,0 +1,249 @@
+
```
Comment (driskull) [inline src/demos/shell/demo-app-temp.css:1]: this file will be removed right?

Comment (asangma) [reply src/demos/shell/demo-app-temp.css:1]: Can do.

File: src/components/calcite-shell-panel/calcite-shell-panel.scss (hunk 1 above)
Comment (asangma) [reply src/components/calcite-shell-panel/calcite-shell-panel.scss:29]: Thanks for catching that.

Comment (asangma) [reply src/components/calcite-shell-panel/calcite-shell-panel.scss]: 🧹

File: src/demos/shell/demo-app-temp.css (hunk 2 above)
Comment (asangma) [reply src/demos/shell/demo-app-temp.css]: 🧹

**Comment by asangma** [comment]
PULLING THE MURGLE TRIGGER!

**Comment by driskull** [comment]
do it!!!

**Comment by AdelheidF** [comment]
I thought we wanted to do this for beta 6, or is this something else?

**Comment by driskull** [comment]
@asangma I think this can be merged now. Is that correct?

**Pull request merged**